### ISSUE TITLE
False positive when employees forward phishing email to ticketing system.

### ISSUE THREAD

**Issue by matthewtwc** [gophish/gophish#2853]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.11.0

Brief description of the issue: This month I made a phishing campaign that included a fake link to a google sheet. Some people usually forward the email to our IT's ticketing system to ask us if the phishing email is legit. When I look at all the tickets we received, all the people have "clicked" on the link provided in the email, but they haven't. 

What are you expecting to see happen? : The email should not say "clicked link" if the person only forwarded the email to us. 

What are you seeing happen? False positives

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by atheriqbal** [comment]
@Steltek is correct. Make sure your IP is whitelisted. Nothing is scanning the emails or sandboxing them.

**Comment by glennzw** [comment]
Yup @Steltek and @atheriqbal are spot on; some appliance is scanning the links in the email. For more info see:

https://github.com/gophish/gophish/issues/2647
https://github.com/gophish/gophish/issues/2646#issuecomment-1330976891
